stripe-core/src/main/java/com/stripe/android/core/exception/GenericStripeException.kt

```diff
@@ -0,0 +1,16 @@
+package com.stripe.android.core.exception
+
+import androidx.annotation.RestrictTo
+
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+class GenericStripeException(
+    cause: Throwable,
+    private val analyticsValue: String? = null,
```

Comment:
Should we require a value here then?

Reply:
Allowing the default to unknown seems good to me. I image a future where we have specific places where we want to use this, and provide a specialized analytics value. But I'd also be happy to remove it as a parameter for now if you'd prefer.